src/Microsoft.Docs.Template/Views/Template/BuildItemListPartial.cshtml

```diff
@@ -124,18 +124,18 @@ else if (Model.Style == "cards")
                             @if (listItem.Image?.Src != null)
                             {
                                 <img class="cardImage" role="presentation" src="@listItem.Image.Src">
-                                <div class="cardText">
-                                    <h3>@listItem.Title</h3>
-                                    @if (!string.IsNullOrEmpty(listItem.Text))
-                                    {
-                                        <p>@listItem.Text</p>
-                                    }
-                                    else if (!string.IsNullOrEmpty(listItem.Html))
-                                    {
-                                        <p>@Html.Raw(removeTagsAndComments(listItem.Html))</p>
-                                    }
-                                </div>
                             }
+                            <div class="cardText">
```

Comment:
indent is different here

Reply:
from the html i think the indent is correct. And i change the indent of <img> to let it looks pretty.